polaris-core/src/main/java/org/apache/polaris/core/persistence/transactional/PolarisMetaStoreManagerImpl.java

```diff
@@ -1966,9 +1966,12 @@ private PolarisEntityResolver resolveSecurableToRoleGrant(
             },
             Function.identity());
 
+    List<PolarisBaseEntity> loadedTasks = new ArrayList<>();
```

Comment:
This `loadTasks` method appears to be called only from test code... Is that expected?

Reply:
I think @collado-mike had some plans for this. I believe it's what's needed next to have better than "best-effort in-memory attempt at cleanup of tables after purge".

For now I'm just trying to have the tests passing while implementing the NonTransactionalMetaStoreManager that also needs to support `loadTasks` to pass the base MetaStoreManager test :)

Reply:
On the plus side, task-leasing happens to be the most intensive concurrency set of unittests we have, so this TaskEntity stuff was what helped discover the missing pieces and bugs in my `NonTransactionalMetaStoreManager` implementation